### DISCUSSION TITLE
bu token nedir?

### DISCUSSION THREAD

**Discussion by kayra533** [IBAX-io/go-ibax#681] [Rewards]
bu tokenden para kazanabilecekmiyiz tam anlamadım??

**Comment by Eemreecan** [comment]
airdrop yapılacak şuan bir yatırım yapmadan puan topluyorsun gelmezse bir şey üzmez ama gelirse sevindirir. Kendi blok zinciri olması çok iyi bu projenin

**Comment by frkselek** [comment]
Bir çeşit L1 mainnet anladığım kadarıyla
